src/create-store.js

```diff
@@ -1,16 +1,19 @@
 import { compose, createStore } from 'redux'
 import reduxLocalStorage from 'redux-localstorage'
+import { identity } from 'lodash'
 
 import rootReducer from './reducers'
 
+const persistState = !process.env.NODE_ENV || process.env.NODE_ENV === 'development'
```

Comment:
Wouldn't be simpler to use `electron-is-dev` as in other parts of the code?

Reply:
That's true for `metronome-wallet-desktop`, right?

Reply:
Hahaha! You are right!